### PR TITLE
WEBCORE-121: Update Changelog and Package version

### PR DESCRIPTION
**Pull request by kimc0de**
Update changelog.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
+## [1.0.1] - 23.08.2022
```
Comment (nitzanashi) [inline CHANGELOG.md:6]: Then update Package.json as well if you already assign the Version

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "output",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (nitzanashi) [inline package.json:3]: And package.lock...

Comment (kimc0de) [reply package.json:3]: yes am doing it from terminal but it added so many packages.. im wondering if it's correct or not

Comment (kimc0de) [reply package.json:3]: in the current package-lock we only have `dependencies: {...}`, running npm i would add `packages: {...}`

Comment (nitzanashi) [reply package.json:3]: Expected

**Review by nitzanashi — APPROVED** [review]

**Comment by kimc0de** [comment]
@z-merge it

**Pull request merged**